py/desispec/joincosmics.py

```diff
@@ -15,7 +15,6 @@
 from desispec.maskbits import ccdmask
 from desispec.maskbits import specmask
 
-from skimage.transform import probabilistic_hough_line
 from skimage.morphology import binary_dilation, binary_closing
```

Comment:
Is the equivalent functionality available within scipy.ndimage, or are the underlying algorithms simple enough that we could code them ourselves?  I'd like to avoid a new dependency on skimage if at all possible.  This is the reason for the Travis test failures.

Reply:
I missed this message, sorry. I'll look into this later today. The function should be simple enough that it's ~an hour of work.

Reply:
@sbailey, the dependency issue is fixed and I checked that the output looks ok, so this should be ready for tests. Sorry for not paying attention to the buildbot, I should know better...